roles/download/defaults/main.yml

```diff
@@ -470,17 +470,17 @@ addon_resizer_image_repo: "{{ kube_image_repo }}/addon-resizer"
 addon_resizer_image_tag: "{{ addon_resizer_version }}"
 
 csi_attacher_image_repo: "{{ quay_image_repo }}/k8scsi/csi-attacher"
-csi_attacher_image_tag: "v1.2.1"
+csi_attacher_image_tag: "v2.2.0"
 csi_provisioner_image_repo: "{{ quay_image_repo }}/k8scsi/csi-provisioner"
-csi_provisioner_image_tag: "v1.3.0"
+csi_provisioner_image_tag: "v1.5.0"
 csi_snapshotter_image_repo: "{{ quay_image_repo }}/k8scsi/csi-snapshotter"
-csi_snapshotter_image_tag: "v1.2.0"
+csi_snapshotter_image_tag: "v2.1.0"
```

Comment:
2.1.1 is released, maybe this could be change before merging (you'll need to rebase master or new pipeline will fail)

Reply:
I updated the snapshotter to 2.1.1 and rebased from master :) CI is currently running